pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java

```diff
@@ -139,11 +140,19 @@ public synchronized void resumeTaskQueue(@Nonnull String taskType) {
    *
    * @param taskType Task type
    */
-  public synchronized void deleteTaskQueue(@Nonnull String taskType) {
+  public synchronized void deleteTaskQueue(@Nonnull String taskType)
+      throws InterruptedException {
     String helixJobQueueName = getHelixJobQueueName(taskType);
     LOGGER.info("Deleting task queue: {} for task type: {}", helixJobQueueName, taskType);
-    // NOTE: set force delete to true to remove the task queue from ZooKeeper immediately
-    _taskDriver.delete(helixJobQueueName, true);
+    // NOTE: set force delete to true to remove the task queue from ZooKeeper immediately.
```

Comment:
The comments here don't see to make sense. Can you remove them?

Reply:
Done.